README.md

```diff
@@ -89,6 +89,7 @@ reported a the [GibHub repository](https://github.com/dotnet-project-file-analyz
 * [**Proj0035** Remove deprecated RestoreProjectStyle property](rules/Proj0035.md)
 * [**Proj0036** Remove None when redundant](rules/Proj0036.md)
 * [**Proj0037** Exclude runtime when all assets are private](rules/Proj0037.md)
+* [**Proj0038** Specify NoWarn rule IDs fully](rules/Proj0038.md)
```

Comment:
I'd be in favour of changing this message slightly to hoist the `fully` to the front of the sentence, since this, while also correct, does not really read very nicely in my opinion. Maybe @LauraXiulan can also weigh in on this.

Reply:
Yes, I would say: Fully specify NoWarn rule IDs